website/snippets/_cloud-environments-info.md

```diff
@@ -42,6 +42,13 @@ For improved reliability and performance on your job runs, you can enable dbt Cl
 
 dbt Cloud caches your project's Git repo after each successful run and retains it for 8 days if there are no repo updates. It caches all packages regardless of installation method and does not fetch code outside of the job runs. 
 
+dbt Cloud will use the cached copy of your project's Git repo under these circumstances:
+
+- Outages from third-party services (for example, the [dbt package hub](https://hub.getdbt.com/)).
```

Comment:
i think it is helpful to have another example, on top of your git one. so adding the git bc that's def a big example that would be helpful for users to know

```suggestion
- Outages from third-party services (for example,  your Git provider or the [dbt package hub](https://hub.getdbt.com/)).
```

Reply:
the Git example is actually mentioned in the screenshot so we're good there